src/_common/popper/popper.ts

```diff
@@ -300,6 +298,23 @@ export default class AppPopper extends Vue {
 		document.addEventListener('click', this.onClickAway, true);
 	}
 
+	private destroyPopper() {
+		this.clearHideTimeout();
+
+		// Clean up any remaining popper elements and instances
+		if (document.body.contains(this.$refs.popper)) {
```

Comment:
You might be able to just do this.$refs.popper.remove() so that we don't ahve to check the body.contains. I'm just worried that the contains call could be slow: https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove Thoughts?

Reply:
That sounds cleaner to me. Changed to check if `this.$refs.popper` exists, and if so then call `remove()`.